crate2nix/src/test.rs

```diff
@@ -129,7 +129,7 @@ impl<'a> PackageAndNode<'a> {
     }
     pub fn version_and_package_id(&mut self, version: &str) -> &mut Self {
         let p = self.get_mut_package();
-        p.version = semver::Version::parse(version).expect("version incorrect");
+        p.version = Version::parse(version).expect("version incorrect");
```

Comment:
anyhow, with_context etc seems to be solved around here

Reply:
Error handling semantics in this function is unchanged, this is just a trivial adjustment required by a dependency update.